src/i18n/i18n.service.ts

```diff
@@ -107,9 +113,14 @@ export class Overridable {
 	 */
 	override(value: string | Observable<string>) {
 		this.isOverridden = true;
+		// To ensure that there are not multiple subscriptions created for the same observable, we
+		// unsubscribe if a subscription already exists for an observable before creating a new one.
+		if (this.subscription) {
+			this.subscription.unsubscribe();
```

Comment:
`subscription.unsubscribe` doesn't appear to be setting the subscription to a falsy value so maybe
```suggestion
			this.subscription.unsubscribe();
			this.subscription = null;
```
to make sure we enter here repeatedly?